src/pybindings/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.1)
+
+if(NOT FAST_DOWNWARD_MAIN_CMAKELISTS_READ)
+    message(
+        FATAL_ERROR
+        "Run cmake on the CMakeLists.txt in the 'src' directory, "
+        "not the one in 'src/search'. Please delete CMakeCache.txt "
+        "from the current directory and restart cmake.")
+endif()
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+## == Project ==
+
+# TODO: Does that need to be project(pydownward) or project(downward)?
+project(pydownward)
+
+# TODO: use multi-line strings to break up the long string when we switch to CMake 3.
+# https://stackoverflow.com/questions/7637539/how-to-split-strings-across-multiple-lines-in-cmake
+option(
+  USE_GLIBCXX_DEBUG
+  "Enable the libstdc++ debug mode that does additional safety checks. (On Linux systems, g++ and clang++ usually use libstdc++ for the C++ library.) The checks come at a significant performance cost and should only be enabled in debug mode. Enabling them makes the binary incompatible with libraries that are not compiled with this flag, which can lead to hard-to-debug errors."
+  FALSE)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+ 
+# TODO: Can you use the same flags here as in search/CMakeLists.txt?
```

Comment:
We should keep all the setup of flags in the general part, so it should not be needed here.